packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -151,11 +151,11 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 		() => ( {
 			...pick( settings, [
```

Comment:
Using `pick` here is error prone as, when one adds a new editor setting, one has to also remember to add the setting name here. I wonder if flipping this to be `omit` would be better?

Reply:
Thanks, @noisysocks

I was getting desperate and then came across your comment.

Maybe we should document this for other contributors. Or use `pick( settings, Object.keys( SETTINGS_DEFAULTS ) )`.